Automattic/wp-calypso
# Magic Login: Reduxify auth handling side of the UI

Accompanies Magic Login redux work done in #12732 & #12880, this is the "auth handler" / interstitial side of things.

## Manual Testing

* Apply code-D5115 to aid in manual testing
* Visit the [login page](http://calypso.localhost:3000/login)
* Click the `Email me a login link` link
* Follow the flow -- it should work (NOTE: 2fa-enabled accounts are not currently supported)
* Try to reuse a link (or modify a fresh one) -- it should not work & you should see the "expired" page

## Automated Testing
`npm run test-client -- --grep "state (login|selectors)"`

For item `Initiate state changes via Redux actions` in #12677
